### ISSUE TITLE
If one want to retrieve raw metric data is there a faster way than using none aggregator ?

### ISSUE THREAD

**Issue by dilipdevaraj-sfdc** [OpenTSDB/opentsdb#1133]
If I have a metric that has a huge tag cardinality - 10,000
When I query using tsdb aggregaros it is fairly fast

-1h:tsdbMetric{tagk1=*,tagk2=*,tagk3=*}:avg
-1h:tsdbMetric{tagk1=*,tagk2=*,tagk3=*}:max
-> this query takes 3sec

However if I query using the none aggregator 
-1h:tsdbMetric{tagk1=*,tagk2=*,tagk3=*}:none
-> this query takes over 10sec

We want to retrieve raw metric data. But for certain metrics which have high tag cardinality and if the time range is increased the performance using none aggregator worsens.

**Comment by dilipdevaraj-sfdc** [comment]
-2d:tsdbMetric:none:1h-sum
This query does not return metrics even after more than 5 minutes.
The metric has a high tag cardinality - order of 10000s

Is there a better way of doing it ? 
Note - I tried using downsampling with none aggregator

**Comment by manolama** [comment]
Nope, None is the best way for now, or use 3.0 without any kind of filters or operators :)
